code/modules/clothing/gloves/color.dm

```diff
@@ -94,7 +94,6 @@
 	name = "fingerless insulated gloves"
 	icon_state = "yellowcut"
 	inhand_icon_state = "ygloves"
-	transfer_prints = TRUE
```

Comment:
All of these gloves that previously transferred prints need to be changed to not cover hands

Reply:
mothblocks says that we should give it a trait instead in a review under this one

Reply:
trait seems fine to